Framework/CurveFitting/src/Functions/ConvTempCorrection.cpp

```diff
@@ -0,0 +1,41 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#include "MantidCurveFitting/Functions/ConvTempCorrection.h"
+#include "MantidAPI/FunctionFactory.h"
+#include <cmath>
+
+namespace {
+constexpr double CONVERSIONFACTOR = 11.606;
+} // namespace
+
+namespace Mantid {
+namespace CurveFitting {
+namespace Functions {
```

Comment:
Also again, don't worry about changing this, but you can now do this instead with C++17:
`namespace Mantid::CurveFitting::Functions {`